pkg/artifactory/datasource_artifactory_file.go

```diff
@@ -0,0 +1,202 @@
+package artifactory
+
+import (
+	"context"
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"github.com/atlassian/go-artifactory/v2/artifactory"
+	"github.com/atlassian/go-artifactory/v2/artifactory/v1"
+	"github.com/hashicorp/terraform/helper/schema"
+	"io"
+	"os"
+)
+
+func datasourceArtifactoryFile() *schema.Resource {
```

Comment:
Nit: this should be `dataSourceArtifactoryFile`

Reply:
Fixed with 29c8eeb